docs/version-1.4.0.md

```diff
@@ -0,0 +1,78 @@
+# Version 1.4.0
+
+**Release Name**: CTK 1.4.0
+
+**Upgrade From**: CTK 1.3.0
+
+**Support**: Stable Release
+
+**Release Date**: 15th April, 2024
+
+## Overview
+
+The 1.4.0 version of CTK includes the following new features:
+
+**New Features in CTK 1.4.0 release** 
+1. Quality Assessment Report is now available for Quality Assessment Collections.
+   This report can be submitted for review just like Compliance Collection report. 
+2. Added a new CTK Landing page
+3. Added Terms & Conditions consent popup for partners on login. If the consent is not given then the partner will be logged out.
```

Comment:
Added Terms & Conditions consent popup for partners during login. If a partner does not provide consent, they will be automatically logged out.